Create mysql_password_expiry.md (#92)
percona-platform/portal-doc: docs/advisors/checks/mysql_password_expiry.md

# MySQL expired passwords

## Description

> MySQL provides password-expiration capability, which enables database administrators to require that users reset their password.  Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,” in which the server permits the client to perform only those operations necessary to reset the expired password.

Reviewer: ```suggestion
MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,”. In this mode, the server allows the client to perform only operations necessary for resetting the expired password.
```

> MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. Once the password is expired, the server either disconnects the client or restricts the client to “sandbox mode,”. In this mode, the server allows the client to perform only operations necessary for resetting the expired password.

Author: ```suggestion
MySQL provides a password-expiration capability, which enables database administrators to require users to reset their passwords. When the password expires, the server either disconnects the client or restricts the client to a sandbox mode. In this mode, the client can only reset the expired password.
```

Such scenarios will impact the application users and this check helps prevent it.

## Resolution

> To resolve this issue, one must change the user account password with [ALTER USER](https://dev.mysql.com/doc/refman/8.0/en/alter-user.html) or [SET PASSWORD](https://dev.mysql.com/doc/refman/8.0/en/set-password.html) command. 

Reviewer: ```suggestion
Change the user account password with [ALTER USER](https://dev.mysql.com/doc/refman/8.0/en/alter-user.html) or [SET PASSWORD](https://dev.mysql.com/doc/refman/8.0/en/set-password.html) command. 
```

> This restores normal access for the session, as well as for subsequent connections that use the account. 

Reviewer 2: This action

> This restores normal access for the session, as well as for subsequent connections that use the account. 

Author: ```suggestion
This action restores normal access for the session, as well as for subsequent connections that use the account. 
```
